### DISCUSSION TITLE
cannot access params global params in layout

### DISCUSSION THREAD

**Discussion by bugineeugine** [vercel/next.js#54982] [App Router]
params cannot be accessed in the layout when it is dynamic, but when it is a page it is accessible
[nextjs.webm](https://github.com/vercel/next.js/assets/61239028/d780c2c0-259b-4d07-bbc6-d0ecbc46c5d2)

**Comment by devjiwonchoi** [comment]
[useSelectedLayoutSegments](https://nextjs.org/docs/app/api-reference/functions/use-selected-layout-segments)

**Reply by bugineeugine** [reply]
iwant access in layout ability example tthis photo in page route

tthis is  page route 
![image](https://github.com/vercel/next.js/assets/61239028/d6a8bb5d-d12a-4e65-91a6-0b36e3a5e5b9)
![image](https://github.com/vercel/next.js/assets/61239028/a282d3ef-a2e3-46bc-8ef2-5f697cd9240f)

this is app route
![image](https://github.com/vercel/next.js/assets/61239028/4ff2cb6c-0719-4403-8e0e-b629d43ef10c)
